hudi-client/hudi-client-common/src/main/java/org/apache/hudi/keygen/KeyGeneratorInterface.java

```diff
@@ -34,8 +33,4 @@
 
   List<String> getRecordKeyFieldNames();
 
-  String getRecordKey(Row row);
```

Comment:
we should make sure there are no backwards incompatible changes to the key generator interface

Reply:
> we should make sure there are no backwards incompatible changes to the key generator interface

Yes, I moved it to `SparkKeyGeneratorInterface`